helm-charts/common/data-prep/README.md

```diff
@@ -38,21 +44,24 @@ Then run the command `kubectl port-forward svc/data-prep 6007:6007` to expose th
 Open another terminal and run the following command to verify the service if working:
 
 ```console
-curl http://localhost:6007/v1/dataprep  \
+curl http://localhost:6007/v1/dataprep/ingest  \
     -X POST \
     -H "Content-Type: multipart/form-data" \
     -F "files=@./README.md"
 ```
 
 ## Values
 
-| Key                    | Type   | Default                 | Description                              |
-| ---------------------- | ------ | ----------------------- | ---------------------------------------- |
-| image.repository       | string | `"opea/dataprep-redis"` |                                          |
-| service.port           | string | `"6007"`                |                                          |
-| REDIS_URL              | string | `""`                    |                                          |
-| TEI_EMBEDDING_ENDPOINT | string | `""`                    |                                          |
-| global.monitoring      | bool   | `false`                 | See ../../monitoring.md before enabling! |
+| Key | Type | Default | Description |
+| --- | ---- | ------- | ----------- |
+
+| service.port | string | `"6007"` | |
+| global.HUGGINGFACEHUB_API_TOKEN | string | `""` | Your own Hugging Face API token |
+| DATAPREP_BACKEND | string | `"REDIS"` | vector DB backend to use, one of "REDIS", "MILVUS" |
+| REDIS_HOST | string | `""` | Redis service URL host, only valid for redis, please see `values.yaml` for other redis configuration |
+| MILVUS_HOST | string | `""` | Milvus service URL host, only valid for redis, please see `values.yaml` for other milvus configuration |
+| TEI_EMBEDDING_ENDPOINT | string | `""` | |
+| global.monitoring | bool | `false` | See ../../monitoring.md before enabling! |
```

Comment:
Table formatting is messed up, there's extra line between headings & content and columns are not aligned.

Reply:
done